cmds/tee/tee.go

```diff
@@ -37,10 +37,8 @@ func copyinput(files []io.Writer, buf []byte) error {
 	return nil
 }
 
-// Parses all the flags and sets variables accordingly
-func handleflags() int {
-	flag.Parse()
-
+// Handle parsed flags and sets variables accordingly
+func handleFlags() int {
```

Comment:
This one was fine, since it's called from main(). My comment, I guess, should actually have been worded "flag.Parse should happen in main instead of init"